monix-execution/shared/src/test/scala/monix/execution/misc/LocalSuite.scala

```diff
@@ -73,8 +75,10 @@ object LocalSuite extends SimpleTestSuite {
     local2 := 100
 
     val f = Local.isolate {
-      local1 := 100
-      Future(local1.get + local2.get)
+      Future {
+        local1 := 100
+        local1.get + local2.get
```

Comment:
it returns updated `local1` (100) + `local2` (999) from below unless we do something use `local2` here

Reply:
fixed both cases